tests/unit/test_transform.py

```diff
@@ -26,7 +31,9 @@ def test_add_to_result_stacks_if_stackable_with_netcdf(
         expected = f"{dstdir}/result.tif"
 
         test_adapter = HarmonyAdapter(
-            Message({"format": {"mime": "application/x-netcdf4"}}), "", None
+            Message({"format": {"mime": "application/x-netcdf4"}}),
+            config=self.config,
+            catalog=None,
```

Comment:
A good next step would be to use a `pystac.Catalog` object here, as Harmony expects. But at least there is a proper `Config` object now.